src/features/content-explorer/content-explorer-modal/ContentExplorerModal.js

```diff
@@ -2,13 +2,15 @@
 import * as React from 'react';
 import classNames from 'classnames';
 
+import type { BreadcrumbProps } from '../../../components/breadcrumb/Breadcrumb';
```

Comment:
I think types are usually imported after components. We'll look into adding this as a prettier configuration.